.github/workflows/deploy-dev.yml

```diff
@@ -41,3 +41,7 @@ jobs:
           --include='assets/***' \
           --exclude='*' \
           ./ ${{env.dest}}
+    - name: Update version number
+      run: |
+          TIMESTAMP = date '+%s'
+          sed -i "s/VERSIONNUMBER/$TIMESTAMP" index.html
```

Comment:
I find it a bit dangerous to replace every `VERSIONNUMBER`. I would be more specific by updating only `"dist/bundle.min.js"`

Reply:
Sorry, I closed the PR a bit too soon, the line will now try to substitute "dist/bundle.min.js?v=VERSIONNUMBER" instead of only VERSIONNUMBER